.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+See [our development docs](https://screed.readthedocs.io/en/latest/dev/).
+
+Be sure to copy and paste the [checklist](https://screed.readthedocs.io/en/latest/dev/coding-guidelines-and-review.html#checklist) in the Pull-Request comment
```

Comment:
With the template setup the checklist should just appear in the comment. Don't know if you want to adjust the text to say that.